bin/z-review-comment

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
```

Comment:
Generally speaking code needs to be put in more functions, some of which could be reusable and moved to a library file instead and exported. I think. Thoughts?